tools/stop_machine_aws.py

```diff
@@ -4,40 +4,28 @@
 import sys
 import logging
 import argparse
+import datetime
 
-def stop(instance, verbose):
+def stop(instance, logger):
     if instance.state['Name'] != 'stopped':
-            if verbose:
-                logger.info("Id: {0}, Platform: {1},Type: {2}, Public IPv4: {3}, AMI: {4}, State: {5}\n".format(instance.id, instance.platform, instance.instance_type, instance.public_ip_address, instance.image.id, instance.state))
             instance.stop()
-            if verbose:
-                logger.info('After stop function:\n')
-                logger.info("Id: {0}, Platform: {1}, Type: {2}, Public IPv4: {3}, AMI: {4}, State: {5}\n".format(instance.id, instance.platform, instance.instance_type, instance.public_ip_address, instance.image.id, instance.state))
+            logger.info("Stop the instance - Id: {0}, Type: {1}, Public IPv4: {2}, State: {3}".format(instance.id, instance.instance_type, instance.public_ip_address, instance.state))
 
-def stop_machines(done_path, type_machine, verbose, argv):
-    print('Stop machines...')
-    ec2 = boto3.resource('ec2', region_name='us-west-2')
-    list_instances = ec2.instances.all()  
-    types=type_machine.split('_')
-    for instance in list_instances:
-        if not instance.state['Name'] == 'stopped' and  instance.instance_type in types:
-            stop(instance, verbose)
-
-    with open(done_path, 'w') as f:
-            f.write(' '.join(argv) + '\n')    
+def stop_machines(type_machine, name_machines, logger):
+    logger.info('Stop machines...')
+    try:
+        ec2 = boto3.resource('ec2', region_name='us-west-2')
+    except:
+        logger.info(f'{datetime.datetime.now()}: Can\'t connect to ec2 with boto3 for stop the machines')
```

Comment:
This should be error log and and not info.
Also in except catch the exception and object and log it